output/outputFormat.go

```diff
@@ -330,10 +331,12 @@ func decode(mediaTypes media.Types, input, output interface{}) error {
 	return decoder.Decode(input)
 }
 
+// BaseFilename returns the base filename of formats.
```

Comment:
I much prefer a lint warning before comments like the above.

Reply:
Agreed.  This is super tedious, and I started getting tired and hasty.  I'll go back through these and double-check my work.